openedx/core/djangoapps/credentials/management/commands/notify_credentials.py

```diff
@@ -81,6 +84,11 @@ def add_arguments(self, parser):
             nargs='+',
             help='Send information only for specific courses.',
         )
+        parser.add_argument(
+            '--program_uuids',
+            nargs='+',
+            help='Send information only for all courses in these programs.',
```

Comment:
Courses or course runs?

Reply:
➕ I agree that it would be helpful to have some clarity on this throughout; it’s taking some digging to remember what exactly is going on here, especially compared to the `--courses` flag.

Reply:
Update coming.